src/src_V2/components/editUserActions/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from '@mui/material';
+import BlockIcon from '@mui/icons-material/Block';
+
+import styles from './styles.module.css';
+
+export default function EditUserActions({
+  onCancel,
+  disabled,
+  allowDeactivate,
+  submitButtonText,
+  onClickDeactivate,
+}) {
+  return (
+    <div className={styles['actions-wrapper']}>
+      <div>
+        {allowDeactivate && (
+          <Button
+            color='error'
+            variant='text'
+            onClick={onClickDeactivate}
+            startIcon={<BlockIcon />}
+          >
+            DEACTIVATE USER
+          </Button>
+        )}
+      </div>
+      <div className={styles['btns-wrapper']}>
+        <Button variant='outlined' onClick={onCancel}>
+          Cancel
+        </Button>
+        <Button
+          type='submit'
+          variant='contained'
+          disabled={disabled}
+          className={styles['submit-btn']}
+        >
+          {submitButtonText || 'Save Changes'}
```

Comment:
why are we reading submit button text from the parent component?

Reply:
because text could differ based on outside component